.github/actions/aws/k8s-versions.yaml

```diff
@@ -0,0 +1,14 @@
+# List of k8s version for GKE tests
+---
+include:
+  - version: "1.23"
+    region: eu-central-1
+  - version: "1.24"
+    region: ap-northeast-1
+  - version: "1.25"
+    region: us-east-2
+    default: true
+  - version: "1.26"
+    region: ca-central-1
+  - version: "1.27"
+    region: eu-north-1
```

Comment:
[Nit, non-blocking] Let's try to make sure all the files end with an empty line for consistency.